### ISSUE TITLE
Small Display cant use toolbar

### ISSUE THREAD

**Issue by Y0ngg4n** [spacedeck/spacedeck-open#200]
## Expected Behavior
I want to use every tool on my laptop

## Actual Behavior
![image](https://user-images.githubusercontent.com/30302975/118981718-f0152b80-b97a-11eb-87e2-ce209b360578.png)

## Possible Solution
Shrink items and Submenus.

## Steps to Reproduce the Problem

  1. Have a small display like a notebook
  2. host your spacedeck
  3. open a spacedeck and try to use it

## Specifications

Latest version

**Comment by rds13** [comment]
May be share, present and bg could be put in a specific menu as submenu.
In your case you might shift to full screen mode it may help.

**Comment by Y0ngg4n** [comment]
@rds13 yeah full screen mode helps but i don´t think this is very user friendly.
